File: ridam krishna/day13/13a.cpp
class Solution {
public:
    vector<vector<int>> generate(int numRows) {
        vector<vector<int>>v1;
        for (int i=0; i<numRows; i++) {
            vector<int>v2(i+1, 1);
            for (int j=1; j<i; j++) {
                v2[j]=v1[i-1][j]+v1[i-1][j-1];
            }
            v1.push_back(v2);
        }
        return v1;
    }
};

File: Mahek Jhamb/Day 13/13A.py
class Solution:
    def generate(self, numRows: int) -> list[list[int]]:
        list1=[]
        for i in range(numRows):
            level=[]
            for j in range(i+1):
                if j==0 or j==i:
                    level.append(1)
                else:
                    level.append(list1[i-1][j-1]+list1[i-1][j])
            list1.append(level)
        return list1